### ISSUE TITLE
Can not reproduced the results reported in the paper

### ISSUE THREAD

**Issue by qiexing** [kenziyuliu/MS-G3D#12]
Thanks for you sharing the code.
I prepared the dataset as in the readme.
And training on the NTU-60 sub dataset.
The scrips i used as: 
dataset=nturgbd-cross-subject
type=joint
python main.py --config ./config/${dataset}/train_${type}.yaml  --device 0 1 --work-dir train_res/${dataset}_${type} 

dataset=nturgbd-cross-subject
type=bone
python main.py --config ./config/${dataset}/train_${type}.yaml  --device 2 3 --work-dir train_res/${dataset}_${type}
But i can only get the result, even after with 4 gpu, batchsize 64。
my      result: joint: 88.68% bone: 89.24% fusion: 90.61%
paper result: joint: 89.40% bone:90.10% fusion: 91.50%

Am I missing something?

**Comment by kenziyuliu** [comment]
Hi @qiexing, thanks for your interest! Couple of things I could think of right now:

- NTU-60 does give relatively large accuracy fluctuations as it's getting saturated, though it is more common to see values near the reported numbers, so this could possibly be a fluctuation. Empirically, I saw min 88.5% max 89.9% for X-Sub Joint over a few runs on different machines, so I would suggest just re-train the model with another seed. It seems NTU-120/Kinetics-400 in contrast gives more stable results.
- The pre-trained models were produced with `--half` and 1 or 2 GPUs (e.g. `--half --batch-size 32 --forward-batch-size 16 --device 0`), so I would suggest trying AMP too.
- Also, empirically the best fusion score does not necessarily come from the best single-stream models, so I would suggest trying a few more models from other epochs for fusion. 

Hope this helps! Feel free to comment below if there's more I could help you with.
